packages/mds-compliance-api/api/index.ts

```diff
@@ -36,3 +37,11 @@ export const api = (app: express.Express): express.Express =>
       ),
       GetComplianceSnapshotIDsHandler
     )
+    .get(
+      pathPrefix('/violation/:violation_id'),
+      checkComplianceApiAccess(
+        scopes => scopes.includes('compliance:read') || scopes.includes('compliance:read:provider')
```

Comment:
Sorry, it's been a while, but what's `compliance:read` vs `compliance:read:provider`? Is the first one for the agency?

Reply:
Yup, one is "read all" access, one is limited to only reading violations which are for the provider_id in the token.